src/components/BalanceSheet/BalanceSheet.tsx

```diff
@@ -23,12 +23,16 @@ export type BalanceSheetViewProps = PropsWithChildren & {
   withExpandAllButton?: boolean
   asWidget?: boolean
   stringOverrides?: BalanceSheetStringOverrides
+  syncWithGlobalDate?: boolean
+  customDateProvider?: boolean
```

Comment:
`syncWithGlobalDate` - to sync local date picker with the global date state
`customDateProvider` - use different DateContext (has to be wrapped with `<DateContext.Provider value={...} />`)